dashboard/15-final/app/ui/dashboard/latest-invoices.tsx

```diff
@@ -1,44 +1,64 @@
 import { LatestInvoice } from '@/app/lib/definitions';
+import { ArrowPathIcon } from '@heroicons/react/24/outline';
+import clsx from 'clsx';
 import Image from 'next/image';
+import { lusitana } from '@/app/ui/fonts.js';
 
 export default async function LatestInvoices({
   latestInvoices,
 }: {
   latestInvoices: LatestInvoice[];
 }) {
   return (
-    <div className="w-full rounded-xl border bg-white p-6 shadow-sm md:col-span-4 lg:col-span-3">
-      <h2 className="font-semibold">Latest Invoices</h2>
-
-      {latestInvoices.map((invoice) => {
-        return (
-          <div
-            key={invoice.id}
-            className="mt-8 flex flex-row items-center justify-between"
-          >
-            <div className="flex items-center">
-              <Image
-                src={invoice.image_url}
-                alt={`${invoice.name}'s profile picture`}
-                className="mr-4 rounded-full"
-                width={32}
-                height={32}
-              />
-              <div className="min-w-0">
-                <p className="truncate text-sm font-semibold md:text-base">
-                  {invoice.name}
-                </p>
-                <p className="hidden text-sm text-gray-500 sm:block">
-                  {invoice.email}
+    <div className="flex w-full flex-col md:col-span-4 lg:col-span-4">
+      <h2 className={clsx(lusitana.className, 'mb-4 text-xl md:text-2xl')}>
+        Latest Invoices
+      </h2>
+      <div className="flex grow flex-col justify-between rounded-xl bg-gray-50 p-4">
+        <div className="bg-white px-6">
+          {latestInvoices.map((invoice, i) => {
+            return (
+              <div
+                key={invoice.id}
+                className={clsx(
+                  i === 0 ? '' : 'border-t',
+                  `flex flex-row items-center justify-between py-4`,
+                )}
```

Comment:
Small suggestion, for readability. When `i !== 0`, add a border-top, is that right?

```suggestion
            className={clsx('flex flex-row items-center justify-between py-4', {
               'border-t': i !== 0,
            })}
```